packable/packable/CHANGELOG.md

```diff
@@ -19,8 +19,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+
+## 0.X.X - 2022-XX-XX
+
 ## 0.2.1 - 2022-02-11
 
+### Changed
+
+- Make byte buffers packing and unpacking more performant.
+
```

Comment:
This version was already released

Reply:
wrong version section. I fixed it in the last commit